docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -400,177 +575,198 @@ The following features are not supported:
 
-The following table lists the supported properties. The visual type is `Mesh` and the visual map is `MeshVisual`.
 
-**Table: Mesh visual properties**
+<!---------------------------------------------------------------------------------------------------------------------------->
+<a name="animatedImageVisual"></a>
+### AnimatedImageVisual
 
-| Property         | Type              | Required          | Description                              |
-|----------------|-----------------|-----------------|----------------------------------------|
-| `ObjectURL`      | `string`            | Yes               | The location of the `.obj` file.         |
-| `MaterialURL`    | `string`            | No                | The location of the `.mtl` file. Leave blank for a textureless object. |
-| `TexturesPath`   | `string`            | If using material | The path to the directory where the textures (including gloss and normal) are stored. |
-| `ShadingMode`    | `integer` or `string` | No                | The type of the shading mode that the mesh uses. |
-| `UseMipmapping`  | `boolean`           | No                | Whether to use mipmaps for textures. By default, `true`. |
-| `UseSoftNormals` | `boolean`          | No                | Whether to average normals at each point to smoother the textures. By default it is `true`. |
-| `LightPosition`  | `Vector3`           | No                | The position, in the stage space of the point light that applies lighting to the model. |
+The animated image visual renders an animated image into the control. 
+Currently, only the GIF format is supported.
 
--   The `ShadingMode` property defines the shading mode type used by the mesh.
 
-    **Table: Shading mode values**
+**Table: AnimatedImageVisual properties**
+
+| Property            | Type               | Required | Description                                                                               |
+|---------------------|--------------------|----------|-------------------------------------------------------------------------------------------|
+| `URL`               | `string`           | Yes, if `URLS` not specified | The URL of the image.                                                 |
+| `URLS`              | `List<string>`     | Yes, if `URL` not specified  | The list of URLS of the animated images.                              |
+| `BatchSize`         | `int`              | No       | The batch size for pre-loading images in the visual (default `1`).                        |
+| `CacheSize`         | `int`              | No       | The cache size for loading images in the visual (default `1`).                            |
+| `FrameDelay`        | `float`            | No       | The number of milliseconds between each frame in the visual (default `0.1`).              |
+| `LoopCount`         | `float`            | No       | The number of times the visual will be looped - unlimited for values < `0` (default `-1`).|
```

Comment:
```suggestion
| `LoopCount`         | `float`            | No       | The number of times the visual will be looped (unlimited for values < `0`) (default `-1`).|

Reply:
and now? 
The number of times the visual will be looped (default `-1`; unlimited for values < `0`).